composer.json

```diff
@@ -29,7 +29,7 @@
         "symfony/polyfill-ctype": "^1.8"
     },
     "require-dev": {
-        "symfony/phpunit-bridge": "^4.4.9|^5.0.9",
+        "symfony/phpunit-bridge": "^4.4.9|^5.0.9|^6.0",
```

Comment:
AFAICT, the bridge does not need to allow different versions, as it has a separate handling (Symfony itself always uses the latest version of that bridge when running tests for its older versions)

Reply:
sure, but it doesn't hurt either :)